LiveSplit.AutoSplitters.xml

```diff
@@ -26019,4 +26019,14 @@
         <Type>Script</Type>
         <Description>Autosplitter/Load Removal available (by Jaffra)</Description>
     </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Frog Legs</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/HandsomeStrife/speedrun-splits-and-autospltters/refs/heads/main/Frog%20Legs/autosplitter.asl</URL>
```

Comment:
Please don't name the file just "autosplitter.asl". A unique identifier for your game would be more clear when the user explores their Components folder.